client/src/banners/active-banner.tsx

```diff
@@ -137,6 +139,36 @@ function PreviewFeaturesBanner({ onDismissed }: { onDismissed: () => void }) {
   );
 }
 
+function MultipleCollectionsBanner({
+  onDismissed,
+}: {
+  onDismissed: () => void;
+}) {
+  const bannerId = BannerId.PREVIEW_FEATURES;
+  const sendCTAEventToGA = useSendCTAEventToGA();
+  const gleanClick = useGleanClick();
+  const onDismissedWithGlean = () => {
+    gleanClick(BANNER_MULTIPLE_COLLECTIONS_DISMISSED);
+    onDismissed();
+  };
+  return (
+    <Banner id={bannerId} onDismissed={onDismissedWithGlean}>
+      <p className="mdn-cta-copy">
+        You can now create multiple collections.{" "}
```

Comment:
I tried the  copy 

"We've added support for Multiple Collections! Check out your **Collections page** to see what's new!"